web/ui/static/js/prom_console.js

```diff
@@ -332,6 +332,7 @@ PromConsole.Graph = function(params) {
 
   this.params = params;
   this.rendered_data = null;
+  // Keep a reference so that further updates (e.g. annotations) can be made.
```

Comment:
There's no updates in the code, so this is still liable to be removed. Mention it's for end users.